content/en/synthetics/multistep.md

```diff
@@ -50,6 +50,8 @@ Multistep API tests can run from Datadog [managed locations][5] and [private loc
 
 Select the **Locations** for your Multistep API test. Multistep API tests can run from both [managed][5] and [private locations][6] depending on your preference for running the test from outside or inside your network.
 
+{{% managed-locations %}} 
+
```

Comment:
I prefer having the {{}} at the end of the paragraph as you wrote in this page  after the mention of PL (multistep).
What do you think?

Reply:
Works for me!